src/container/srv_target.c

```diff
@@ -2548,39 +2667,54 @@ ds_cont_tgt_ec_eph_query_ult(void *data)
 					}
 				}
 
-				if (!is_failed_tgts)
-					min_eph = min(min_eph, ec_eph->ce_ephs[i]);
+				if (!is_failed_tgts) {
+					min_ec_agg_eph = min(min_ec_agg_eph,
+							     ec_eph->cte_ec_agg_ephs[i]);
+					min_stable_eph = min(min_stable_eph,
+							     ec_eph->cte_stable_ephs[i]);
+				}
 			}
 
-			if (min_eph == 0 || min_eph == DAOS_EPOCH_MAX ||
-			    min_eph <= ec_eph->ce_last_eph) {
-				if (min_eph > 0 && min_eph < ec_eph->ce_last_eph)
-					D_ERROR("ignore for now "DF_X64" < "DF_X64
-						" "DF_UUID"\n", min_eph, ec_eph->ce_last_eph,
-						DP_UUID(ec_eph->ce_cont_uuid));
+			if (min_ec_agg_eph == 0 || min_ec_agg_eph == DAOS_EPOCH_MAX ||
+			    min_stable_eph == 0 || min_stable_eph == DAOS_EPOCH_MAX ||
+			    (min_ec_agg_eph <= ec_eph->cte_last_ec_agg_epoch &&
+			     min_stable_eph <= ec_eph->cte_last_stable_epoch)) {
+				if (min_ec_agg_eph > 0 && min_stable_eph > 0 &&
+				    min_ec_agg_eph <= ec_eph->cte_last_ec_agg_epoch &&
+				    min_stable_eph <= ec_eph->cte_last_stable_epoch)
```

Comment:
Why EC_agg_epoch/stable_epoch cannot keep unchanged between two cycle checks? I think that is normal instead or ERROR().

Reply:
The D_ERROR() is just kept from original code, commonly should not get smaller epoch,  but better to change the <= to < same as before.